pkgpanda/test_check.py

```diff
@@ -1,8 +1,17 @@
 from subprocess import check_output, PIPE, Popen, STDOUT
 
-from pkgpanda.util import resources_test_dir
+from pkgpanda.util import is_windows, resources_test_dir
 
-list_output = """WARNING: `not_executable.py` is not executable
+if is_windows:
+    list_output = """pkg1--12345
+ - hello_world_ok.py
+ - not_executable.py
+pkg2--12345
+ - failed_check.py
+ - shell_script_check.sh
+"""
+else:
+    list_output = """WARNING: `not_executable.py` is not executable
```

Comment:
Instead of having tests behave differently when they're running on Windows, I'd rather if we set it up so that we ran a different set of tests on Windows vs when we're on Unix. We can accomplish this by defining test functions for Windows separately from test functions intended for Unix and using `pytest`[1] to skip them based on whether we're on the right platform. The advantage to this is that the individual tests are simpler to understand, platform-specific logic is separated into different functions, and it's easy to tell when they've run. If Unix-specific test code is in its own function that gets skipped on Windows, `pytest` will report that it was skipped, and I can then ignore it when trying to investigate a failed Windows test run.

Something like this:

```
def helper():
    pass

@pytest.mark.skipif(is_windows)
def test_unix():
    helper()
    # Unix stuff here
    

@pytest.mark.skipif(not is_windows)
def test_windows():
    helper()
    # Windows stuff here

def test_all_platforms():
    pass
```

[1] https://docs.pytest.org/en/latest/skipping.html#skipping-test-functions